### PR TITLE
Explicitly turn off MLIR solvers for performance comparison

### PR DESCRIPTION
**Pull request by yiqian1**

### CODE REVIEW

File: mlir/utils/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -376,7 +376,7 @@ pipeline {
                            steps {
                                dir('MIOpen') {
                                    getAndBuildMIOpen("--prefix ${WORKSPACE}/MIOpen/MIOpenDeps",
-                                                     '''-DMIOPEN_BACKEND=HIP
+                                                     '''-DMIOPEN_BACKEND=HIP -DMIOPEN_USE_MLIR=Off
```
Comment (krzysz00) [inline mlir/utils/jenkins/Jenkinsfile:379]: I think cmake booleans are usually spelled in all caps, so `OFF` here is good style - or, in the alternative, `0`

**Review by krzysz00 — APPROVED** [review]
LGTM mod one style nit that doesn't strictly need to be resolved

**Comment by whchung** [comment]
@yiqian1 Can we add one more column in the performance report to show: MIOpen (with tuned MLIR kernel) / MIOpen (without MLIR)?

**Comment by yiqian1** [comment]
> @yiqian1 Can we add one more column in the performance report to show: MIOpen (with tuned MLIR kernel) / MIOpen (without MLIR)?

Yes. Will do.

**Pull request merged**